cinema_service/settings.py

```diff
@@ -47,6 +36,7 @@
     "debug_toolbar",
     "cinema",
     "user",
+    "drf_spectacular",
```

Comment:
I recommend listing external packages first in the INSTALLED_APPS and then your own apps.